tests/pr_test.go

```diff
@@ -54,6 +55,37 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "advanced-key-protect",
+		TarIncludePatterns: []string{
+			"*.tf",
```

Comment:
you need to also add the example directory (aka `advancedExampleTerraformDir`)

Reply:
@ocofaigh done